dev/devicelab/bin/tasks/service_extensions_test.dart

```diff
@@ -81,7 +81,7 @@ void main() {
 
       final Future<VMExtensionEvent> navigationFuture = navigationEvents.first;
       // This tap triggers a navigation event.
-      device.tap(100, 100);
+      device.tap(100, 200);
```

Comment:
This was high enough on my display to cause the settings drawer to be triggered. Bumping this value to tap further down the screen fixes the failure.